tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -5189,38 +5205,70 @@ <h2>Show similarity to selected datapoint</h2>
         });
       },
 
+      /**
+       * Handler for closing the partial dependence plots holder for a feature.
+       */
+      hidePdElement_: function(featureContainer) {
+        // Clear the plots
+        const element = featureContainer.querySelector('.pd-entry-container');
+        this.clearPdElementCharts_(element);
+        element.setAttribute('hidden', true)
+        // Handle icon
+        const icon = featureContainer.querySelector('.pd-row-arrow');
+        icon.classList.remove('normal-icon');
+        icon.classList.add('rotated-icon');
+        // Handle row style
+        const row = featureContainer.querySelector('.perf-table-entry');
+        row.classList.add('perf-table-row');
+        row.classList.remove('perf-table-row-expanded');
+      },
+
+      showPdElement_: function(featureContainer) {
+        const element = featureContainer.querySelector('.pd-entry-container');
+        element.removeAttribute('hidden')
+        // Handle icon
+        const icon = featureContainer.querySelector('.pd-row-arrow');
+        icon.classList.add('normal-icon');
+        icon.classList.remove('rotated-icon');
+        // Handle row style
+        const row = featureContainer.querySelector('.perf-table-entry');
+        row.classList.remove('perf-table-row');
+        row.classList.add('perf-table-row-expanded');
+      },
+
       hideAllPdElements_: function() {
-        const elements = Polymer.dom(this.root).querySelectorAll('.tf-category-pane-content');
+        const elements = Polymer.dom(this.root).querySelectorAll('.feature-container');
         for (let i = 0; i < elements.length; i++) {
           this.hidePdElement_(elements[i]);
         }
       },
 
+      pdInputChanged: function(event) {
+        // Debounce a refresh request, for when multiple updates are happening
+        // in succession.
+        clearTimeout(this.refreshPDPlotTimer_);
+        this.refreshPDPlotTimer_ = setTimeout(() => this.pdInputChangedImpl(event), 500);
```

Comment:
prefer to use `this.debounce` https://polymer-library.polymer-project.org/1.0/api/classes/Polymer.Base#method-debounce

Reply:
Done! Thank you for this, I wasn't aware of the native option.